server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/SessionRoutesContract.scala

```diff
@@ -132,6 +133,7 @@ object SessionRoutesContract {
                          |    "urn:ietf:params:jmap:mdn": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6"
                          |  },
                          |  "username" : "bob@domain.tld",
+                         |  "delegatedUsers": [],
```

Comment:
No we should not add a custom property to the session.

We should add new entries into the `accounts` field.